common/app/views/fragments/atoms/structureddata/recipe.scala.html

```diff
@@ -13,26 +15,26 @@
           "@@type": "Person",
           "name": "@name"
         },
-      } 
-      @recipe.atom.contentChangeDetails.published.map { published => "datePublished": "@published",}
+      }
+      @recipe.atom.contentChangeDetails.created.map { created => "dateCreated": "@{new DateTime(created.date).toString("yyyy-MM-dd'T'HH:mm:ssZ")}",}
+      @recipe.atom.contentChangeDetails.published.map { published => "datePublished": "@{new DateTime(published.date).toString("yyyy-MM-dd'T'HH:mm:ssZ")}",}
+      @recipe.atom.contentChangeDetails.lastModified.map { modified => "dateModified": "@{new DateTime(modified.date).toString("yyyy-MM-dd'T'HH:mm:ssZ")}",}
+
+      "description": "a tasteful @{recipe.data.tags.category.headOption.getOrElse("recipe")} with @{recipe.data.ingredientsLists.flatMap(_.ingredients).headOption.map(_.item).getOrElse("secrets ingredients! You need to know about them")}",
      
      @* 
      TODO - We do not have this description and rating in the model yet  
-     "description": "Mango, strawberries, and sweetened dried cranberries are a vibrant addition to mixed greens tossed with an oil and balsamic vinegar dressing.",
+     
      "aggregateRating": {
        "@@type": "AggregateRating",
        "ratingValue": "5",
        "reviewCount": "52"
      },
      *@
-
-     @recipe.data.time.preparation.map {minutes => "prepTime": "@formatDuration(minutes)",}
-     @recipe.data.time.cooking.map {minutes => "cookTime": "@formatDuration(minutes)",}
-     
-     @* TODO - @recipe.data.time.total.map {minutes => "totalTime": "@formatDuration(minutes)",}  *@
-
-     @recipe.data.serves.map { serves => "recipeYield": "@yieldValue(serves)",}
-
+      @recipe.data.time.preparation.map {minutes => "prepTime": "PT@{minutes}M",}
+      @recipe.data.time.cooking.map {minutes => "cookTime": "PT@{minutes}M",}
+      @recipe.data.time.preparation.combine(recipe.data.time.cooking).map { minutes => "totalTime": "PT@{minutes}M",}
```

Comment:
Unfortunately, we can't use `|+|` as `twirl` does not seems to correctly recognize the short operator. As there is only `2` elements to sum it does not hurt the readability.